.localization-config

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
This seems to be a new file in the root. Is that intentional? Does it affect the EF Content as well? 

Just checking, I am not familiar with the details of the docs system.